tests/Unit/Http/GraphQLHandlerTest.php

```
@@ -24,6 +30,100 @@
+    public function itReturnsGraphQlErrorExtensionsWhenAnInternalFailureOccursBeforeExecution(): void
+    {
+        // Arrange
+        [$handler] = $this->createHandler();
+        $schemaPath = dirname(__DIR__, 3) . '/src/GraphQL/Schema/schema.graphql';
+        $backupPath = $schemaPath . '.' . uniqid('backup-', true);
+
+        self::assertFileExists($schemaPath);
+        self::assertTrue(rename($schemaPath, $backupPath));
+        set_error_handler(
+            static function (int $severity, string $message, string $file, int $line): never {
+                throw SchemaFileAccessWarning::fromPhpWarning($message, $file, $line, $severity);
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'App\Tests\Unit\Http\SchemaFileAccessWarning' in method 'itReturnsGraphQlErrorExtensionsWhenAnInternalFailureOccursBeforeExecution'. (undefined)

(StaticAccess)